resources/views/ItemList/Components/Filter/ItemDropdown.twig

```diff
@@ -1,17 +1,17 @@
-<script type="x/template" id="vue-item-dropdown">
+<script type="x/template" id="vue-findologic-item-dropdown">
     <div class="fl-dropdown">
         <div v-for="value in facet.values.slice(0, facet.itemCount)" class="form-check">
-            <input :id="'option-' + value.id" :disabled="isLoading" :checked="value.selected"  @change="selected(value.name)" class="form-check-input hidden-xs-up" type="checkbox"/>
-            <label :for="'option-' + value.id" class="form-check-label">${ value.name }</label>
-            <div class="filter-badge" v-if="value.count">${ value.count }</div>
+            <input :id="'option-' + value.id" :disabled="isLoading" :checked="value.selected"  @change="selected(value.name)" class="form-check-input hidden-xs-up" type="checkbox" style="display:none"/>
+            <label :for="'option-' + value.id" class="form-check-label" v-text="value.name"></label>
+            <div class="filter-badge" v-if="value.count" v-text="value.count"></div>
         </div>
         <div v-if="facet.values.slice(facet.itemCount, facet.values.length).length" class="fl-dropdown-container custom-select" tabindex="0" @click="toggleDropdown()" @blur="hideDropdown()">
             <span class="fl-dropdown-label">{{ trans("Findologic::Template.pleaseSelect") }}</span>
             <ul v-show="isShowDropdown" class="fl-dropdown-content form-check">
                 <li v-for="value in facet.values.slice(facet.itemCount, facet.values.length)" @click="selected(value.name)" class="fl-dropdown-item" :class="{'form-check-label': !value.selected}">
-                    <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" :disabled="isLoading">
-                    <label :for="'option-' + value.id" :class="{'form-check-label': value.selected}">${ value.name }</label>
-                    <div class="filter-badge" v-if="value.count">${ value.count }</div>
+                    <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" :disabled="isLoading" style="display:none">
```

Comment:
Shouldn't the class `hidden-xs-up` handle this style?

Reply:
I can't seem to get that class in frontend. It might have been removed in Ceres 5.0. Anyway, updated Findologic SCSS file to include the style.